ingest/api/ingestapi.py

```diff
@@ -88,6 +88,15 @@ def get_link_from_resource(self, resource, link_name):
         links = resource.get('_links', {})
         return links.get(link_name, {}).get('href')
 
+    def get_latest_schema_url(self, high_level_entity, domain_entity, concrete_entity):
+        latest_schema = self.get_schemas(
+            latest_only=True,
+            high_level_entity=high_level_entity,
+            domain_entity=domain_entity.split('/')[0],
+            concrete_entity=concrete_entity
+        )
+        return latest_schema[0]['_links']['json-schema']['href'] if latest_schema else None
```

Comment:
Could it be a case that `self.get_schemas` returns an empty list?
If yes, then this code is going to throw an exception.

Reply:
Thanks, Karel for making sure it doesn't throw an exception. The trailing `if latest_schema else None` should handle the empty list  since `bool([])` evaluates to False.

Reply:
I added a test to reflect that test case